README.md

```diff
@@ -23,23 +23,36 @@ Users should install the dependencies by the following requirments:
 
 For the tensorflow extensions:
 
-* libpng-dev
+* libpng-dev 
+```bash
+sudo aptitude install libpng-dev
+```
 * libjpeg-dev
+```bash
+sudo aptitude install libjpeg-dev
+```
 * libtensorflow.so
-	* Please refer to the TensorFlow website for more details
-	```base
-	RUN wget https://storage.googleapis.com/tensorflow/libtensorflow/libtensorflow-cpu-linux-x86_64-2.4.0.tar.gz \
-		&& tar -C /usr/local -xzf libtensorflow-cpu-linux-x86_64-2.4.0.tar.gz \
-		&& ldconfig
+	* Please refer to [the TensorFlow website](https://www.tensorflow.org/install/lang_c) for more details
+	```bash
+	cd ~
+	wget https://storage.googleapis.com/tensorflow/libtensorflow/libtensorflow-cpu-linux-x86_64-2.4.0.tar.gz
+	sudo tar -C /usr/local -xzf libtensorflow-cpu-linux-x86_64-2.4.0.tar.gz
+	rm -rf libtensorflow-cpu-linux-x86_64-2.4.0.tar.gz
 	```
 * libtensorflowlite\_c.so
 	* You can use the following commands to get libtensorflowlite\_c.so
 	```bash
-	# Download TensorFlow deps
+        # Download TensorFlow deps
```

Comment:
Please remove extra space before the `#`

Reply:
Apologies, that .md should not have been there in that code block, I have removed it now.